sdk/ai/azure-ai-projects/azure/ai/projects/aio/operations/_patch.py

```diff
@@ -1417,10 +1433,11 @@ async def create_stream(
         truncation_strategy: Optional[_models.TruncationObject] = None,
         tool_choice: Optional["_types.AgentsApiToolChoiceOption"] = None,
         response_format: Optional["_types.AgentsApiResponseFormatOption"] = None,
+        parallel_tool_calls: Optional[bool] = None,
         metadata: Optional[Dict[str, str]] = None,
-        event_handler: Optional[_models.AsyncAgentEventHandler] = None,
+        event_handler: _models.BaseAsyncAgentEventHandlerT = _defaultAgentEventHandler,  # type: ignore[assignment]
```

Comment:
You must not use a mutable default value instance (that is actually being mutated)

Reply:
Fixed and pushed already